src/test/java/com/aws/greengrass/smbridge/TopicMappingTest.java

```diff
@@ -6,21 +6,21 @@
 package com.aws.greengrass.smbridge;
```

Comment:
Do you have code coverage reporting enabled on this repo? What are the numbers?

Reply:
![image](https://user-images.githubusercontent.com/85959265/127554575-0e175298-b020-4b1e-b41c-c9501720fe44.png)

Reply:
![image](https://user-images.githubusercontent.com/85959265/127554678-d4992a67-b5ab-4be7-b4f0-f7aa71cff357.png)